source/Calamari.Tests/KubernetesFixtures/Approved/KubernetesContextScriptWrapperFixture.ExecutionWithEKS_IAMAuthenticator.approved.txt

```diff
@@ -6,12 +6,7 @@
 [Verbose] "kubectl" config use-context octocontext --request-timeout=1m
 [Verbose] "kubectl" config set-cluster octocluster --insecure-skip-tls-verify=true --request-timeout=1m
 [Info] Creating kubectl context to <server> (namespace calamari-testing) using EKS cluster name my-eks-cluster
-[Error] The EKS cluster Url specified should contain a valid aws region name
-[Verbose] Unable to authenticate to <server> using the aws cli. Failed with the error: System.ArgumentNullException: Value cannot be null. (Parameter 's')
-   at System.IO.StringReader..ctor(String s)

```

Comment:
We can still log this error, just with the message only. Avoiding the stacktrace line number